Source/driver/sdc_util.F90

```diff
@@ -248,8 +265,22 @@ subroutine sdc_solve(dt_m, U_old, U_new, C, sdc_iteration)
           imode = MF_NUMERICAL_JAC
        endif
 
+       ! relative tolerances
+       rtol(0) = tol_dens
+       rtol(1:nspec_evolve) = tol_spec
+       rtol(nspec_evolve+1) = tol_ener
+
+       ! absolute tolerances
+       atol(0) = sdc_solver_atol * U_old(URHO)
+       atol(1:nspec_evolve) = sdc_solver_atol * U_old(URHO)   ! this way, atol is the minimum x
```

Comment:
I don't follow this choice or the comment -- can you elaborate?

Reply:
we are integrating (rho X), so I need to turn what I want the minimum value of X to track into the atol for (rho X).

Reply:
OK. But since sum(rho X) = rho, isn't this scaling based on X == 1?

Reply:
OK, I get it now.